lib/engine/game/g_18_usa/game.rb

```diff
@@ -119,6 +119,14 @@ class Game < G1817::Game
           },
         ].freeze
 
+        def game_phases
+          phases = self.class::PHASES
+          return phases unless @optional_rules.include?(:seventeen_trains)
+
+          phases = phases.dup
```

Comment:
there's no reason to dup, reject makes a copy

Reply:
Fixed.